KarhooUISDK/Screens/CheckoutScreen/Checkout/DetailsCellViewController.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  DetailsCellViewController.swift
+//  KarhooUISDK
+//
+//  Created by Bartlomiej Sopala on 15/11/2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+class DetailsCellViewController: UIHostingController<DetailsCellView>, DetailsCellViewControllerProtocol {
+    
+    var onClickAction: (() -> Void)!
```

Comment:
We're not using action anymore in new views afaik. Let's rename this to `onTap` maybe?